<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Ranjani" failures="0" tests="16" name="com.tutorialopencart.qa.testcases.LoginTest" time="318.358" errors="1" timestamp="2024-09-19T10:36:21 EDT" skipped="5">
  <testcase classname="com.tutorialopencart.qa.testcases.LoginTest" name="loginValidcredential" time="14.386"/>
  <testcase classname="com.tutorialopencart.qa.testcases.LoginTest" name="loginInvalidcredential" time="0.000">
    <skipped/>
  </testcase> <!-- loginInvalidcredential -->
  <testcase classname="com.tutorialopencart.qa.testcases.LoginTest" name="loginValidcredential" time="0.000">
    <skipped/>
  </testcase> <!-- loginValidcredential -->
  <testcase classname="com.tutorialopencart.qa.testcases.LoginTest" name="loginInvalidPassword" time="14.540"/>
  <testcase classname="com.tutorialopencart.qa.testcases.LoginTest" name="loginInvalidcredential" time="14.977"/>
  <testcase classname="com.tutorialopencart.qa.testcases.LoginTest" name="loginWithoutCredential" time="12.614"/>
  <testcase classname="com.tutorialopencart.qa.testcases.LoginTest" name="loginWithoutCredential" time="0.000">
    <skipped/>
  </testcase> <!-- loginWithoutCredential -->
  <testcase classname="com.tutorialopencart.qa.testcases.LoginTest" name="loginInvalidPassword" time="0.000">
    <skipped/>
  </testcase> <!-- loginInvalidPassword -->
  <testcase classname="com.tutorialopencart.qa.testcases.LoginTest" name="loginValidcredential" time="12.234"/>
  <testcase classname="com.tutorialopencart.qa.testcases.LoginTest" name="setup" time="181.307">
    <error message="Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure. 
Host info: host: &#039;RANJANI&#039;, ip: &#039;192.168.1.222&#039;" type="org.openqa.selenium.SessionNotCreatedException">
      <![CDATA[org.openqa.selenium.SessionNotCreatedException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure. 
Host info: host: 'RANJANI', ip: '192.168.1.222'
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:545)
at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:234)
at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:163)
at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:143)
at com.tutorialopencart.qa.testcases.BaseTest.setup(BaseTest.java:81)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:71)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:400)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:333)
at org.testng.internal.invokers.TestInvoker.runConfigMethods(TestInvoker.java:833)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:600)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1570)
Caused by: org.openqa.selenium.TimeoutException: java.net.http.HttpTimeoutException: request timed out
Build info: version: '4.23.0', revision: '4df0a231af'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '22.0.1'
Driver info: driver.version: RemoteWebDriver
at org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute0(JdkHttpClient.java:389)
at org.openqa.selenium.remote.http.AddSeleniumUserAgent.lambda$apply$0(AddSeleniumUserAgent.java:42)
at org.openqa.selenium.remote.http.Filter.lambda$andFinally$1(Filter.java:55)
at org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute(JdkHttpClient.java:355)
at org.openqa.selenium.remote.tracing.TracedHttpClient.execute(TracedHttpClient.java:54)
at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:89)
at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:75)
at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:61)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:162)
at org.openqa.selenium.remote.TracedCommandExecutor.execute(TracedCommandExecutor.java:53)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:527)
... 28 more
Caused by: java.net.http.HttpTimeoutException: request timed out
at java.net.http/jdk.internal.net.http.ResponseTimerEvent.handle(ResponseTimerEvent.java:63)
at java.net.http/jdk.internal.net.http.HttpClientImpl.purgeTimeoutsAndReturnNextDeadline(HttpClientImpl.java:1788)
at java.net.http/jdk.internal.net.http.HttpClientImpl$SelectorManager.run(HttpClientImpl.java:1385)
]]>
    </error>
  </testcase> <!-- setup -->
  <testcase classname="com.tutorialopencart.qa.testcases.LoginTest" name="loginInvalidUsername" time="0.000">
    <skipped/>
  </testcase> <!-- loginInvalidUsername -->
  <testcase classname="com.tutorialopencart.qa.testcases.LoginTest" name="loginWithoutCredential" time="11.378"/>
  <testcase classname="com.tutorialopencart.qa.testcases.LoginTest" name="loginInvalidPassword" time="14.079"/>
  <testcase classname="com.tutorialopencart.qa.testcases.LoginTest" name="loginInvalidUsername" time="14.065"/>
  <testcase classname="com.tutorialopencart.qa.testcases.LoginTest" name="loginInvalidUsername" time="15.352"/>
  <testcase classname="com.tutorialopencart.qa.testcases.LoginTest" name="loginInvalidcredential" time="13.426"/>
</testsuite> <!-- com.tutorialopencart.qa.testcases.LoginTest -->
